<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Gaëtan Benoit</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <meta name="viewport" content="width=device-width">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <div class="greeting">
        <h1>Gaëtan Benoit</h1>
        <h2>Postdoctoral researcher in bioinformatics</h2>
        <img src="images/IMG_20220316_135238.jpg" alt="Logo">
        Email: gaetanbenoitdev at gmail.com
        
        <p>Genscale, Inria Rennes (France)</p>
        
        <ul>
          <li><a href="https://scholar.google.com/citations?user=EM12sfgAAAAJ&hl=fr&oi=ao"><strong>Scholar</strong></a></li>
          <li><a href="https://github.com/GaetanBenoitDev"><strong>Github</strong></a></li>
          <li><a href="https://twitter.com/gaetan_benoit"><strong>Twitter</strong></a></li>
        </ul>
        </div>
      </header>



      <section>


        <h1>Software</h1>
        <ul>
          <li><a href="https://github.com/GaetanBenoitDev/metaMDBG">MetaMDBG:</a> fast and low-memory assembly for long and accurate metagenomics reads using multiplex minimizer de-Bruijn graph</li>  
          <li><a href="https://github.com/GATB/simka">Simka:</a> large-scale de novo comparative metagenomics based on efficient k-mer counting of multiple datasets</li>
          <li><a href="https://github.com/GATB/simka">SimkaMin:</a> ultra-fast estimation of jaccard and bray-curtis distances using k-mer skecthes</li>  
          <li><a href="https://github.com/GATB/leon">Leon:</a> read compression based on probabilistic de-Bruijn graph</li>  
          <li><a href="https://github.com/GATB/bloocoo">Bloocoo:</a> memory efficient read correction using a bloom filter</li>  
        </ul>

        <h1>Publications</h1>
        
        <ul>
          <li><a href="https://www.biorxiv.org/content/10.1101/2023.07.07.548136v1">Efficient high-quality metagenome assembly from long accurate reads using minimizer-space de Bruijn graphs.</a> G Benoit, S Raguideau, R James, A M Phillippy, R Chikhi and C Quince; Accepted in <i>Nature Biotechnology</i>, 2023</li>
          <li><a href="https://elifesciences.org/articles/78129">Genomic evidence for global ocean plankton biogeography shaped by large-scale current systems.</a> DJ Richter, R Watteaux, T Vannier, J Leconte, P Frémont, G Reygondeau, N Maillet, N Henry, G Benoit, ..., O Jaillon; <i>Elife</i>, 2022</li>
          <li><a href="https://academic.oup.com/bioinformatics/article/36/4/1275/5559271">SimkaMin: fast and resource frugal de novo comparative metagenomics.</a> G Benoit, M Mariadassou, S Robin, S Schbath, P Peterlongo, C Lemaitre; <i>Bioinformatics</i>, 2020</li>
          <li><a href="https://peerj.com/articles/cs-94/">Multiple comparative metagenomics using multiset k-mer counting.</a> G Benoit, P Peterlongo, M Mariadassou, E Drezen, S Schbath, D Lavenier, C Lemaitre; <i>PeerJ Computer Science</i>, 2016</li>
          <li><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0709-7">Reference-free compression of high throughput sequencing data with a probabilistic de Bruijn graph.</a> G Benoit, C Lemaitre, D Lavenier, E Drezen, T Dayris, R Uricaru, G Rizk; <i>BMC bioinformatics</i>, 2015</li>
        </ul>
        
        <h1>Communications</h1>
        <h2>Talks</h2>
        <ul> 
          <li>JOBIM, Lille, 2017, Simka: Large-scale de novo comparative metagenomics.</li>
          <li>RCAM, Paris, 2015, <a href="https://hal.science/hal-01231795">Fast kmer-based method for estimating the similarity between numerous metagenomic datasets</a></li>
          <li>EBAME, Brest, 2015, Fast kmer-based method for estimating the similarity between numerous metagenomic datasets</li>
        </ul>
        
        <h2>Posters</h2>
	<ul>
          <li>JOBIM, Clermont-Ferrand, 2015, <a href="https://inria.hal.science/hal-01180603"> Fast kmer-based method for estimating the similarity between numerous metagenomic datasets</a></li>
          <li>ECCB, Strasbourg, 2014, <a href="https://hal-emse.ccsd.cnrs.fr/IRISA_SET/hal-01092960v1">Bloocoo, a memory efficient read corrector</a></li>
        </ul>
        
        
        <h1>Other projects</h1>
        <ul>
          <li><a href="https://store.steampowered.com/app/1263650/Eatn_Eaten/">Eat'n eaten:</a> an ecosystem sim game, developed in C# with Unity engine</li>  
        </ul>


      </section>
      <footer>
        <p>This project is maintained by <a href="https://github.com/GaetanBenoitDev">Gaëtan Benoit</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>
  </body>
</html>
